configs/pinia.json

```diff
@@ -0,0 +1,48 @@
+{
+  "index_name": "pinia",
+  "start_urls": [
+    {
+      "url": "https://pinia.esm.dev/introduction.html",
+      "selectors_key": "introduction",
+      "tags": ["Introduction"],
+      "page_rank": 1
+    },
+    {
+      "url": "https://pinia.esm.dev/getting-started.html",
+      "selectors_key": "getting-started",
+      "tags": ["Introduction"],
+      "page_rank": 1
+    },
+    {
+      "url": "https://pinia.esm.dev/core-concepts/",
+      "selectors_key": "core-concepts",
+      "tags": ["Core Concepts"],
+      "page_rank": 10
+    },
+    {
+      "url": "https://pinia.esm.dev/ssr/",
+      "selectors_key": "ssr",
+      "tags": ["SSR"],
+      "page_rank": 8
+    },
+    {
+      "url": "https://pinia.esm.dev/cookbook/",
+      "selectors_key": "cookbook",
+      "tags": ["Cookbook"],
+      "page_rank": 5
+    }
+  ],
+  "stop_urls": [],
+  "selectors": {
+    "lvl0": ".content h1",
+    "lvl1": ".content h2",
+    "lvl2": ".content h3",
+    "lvl3": ".content h4",
+    "lvl4": ".content h5",
+    "lvl5": ".content p, .content li",
+    "text": ".content [class^=language-]"
```

Comment:
I don't know if adding the code makes sense here but it would be nice to be able to search it if it's possible

Reply:
as long as `document.querySelectorAll('yourSelector');` returns what you'd except to be returned from the search, it's good! (and in this case, it is)